pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// ----------------------------
+// #SETTINGS
+// ----------------------------
+@mixin alert-base($alert-color) {
+    border-top: rem(4) solid $alert-color;
+    .alert-icon {
+        color: palette(grayscale, white);
+        background-color: $alert-color;
+    }
+}
+
+// ----------------------------
+// #GENERAL
+// ----------------------------
+.alert {
+    border: rem(1) solid palette(grayscale-cool, light);
+    border-radius: rem(3);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    overflow: auto;
+}
+
+.alert-icon {
+    // create a circle around the icon
+    border-radius: 50%;
+
+    // create room around the icon for the circle
+    padding: rem(7);
+
+    @include susy-media($bp-screen-md) {
+        float: left;
```

Comment:
Do you have an example function/method for doing that?  I think I noticed one somewhere, but have no idea where it is.